lib/src/hooks/useDynamicForm.tsx

```diff
@@ -256,23 +321,24 @@ const DynamicFormItemComponent: React.FC<{
     } = {
         name: props.item?.key || "",
         label: props.item?.label || "",
-        value: (props.state as string) || "",
+        value: props.state as any,
```

Comment:
commonProps value has type of `any`, do we still need `as any` here? maybe can move type for commonProps above?

Reply:
You're right there's n need as it repeats itself, I did this mostly for readability so that it's obvious this value will be mapped as common prop to all components and can really be any value.